understanding/22/consistent-help.html

```diff
@@ -43,49 +43,48 @@ <h2>Consistent Help</h2>
+        <h3>Forms of help</h3>
 
+        <h4>Human contact details</h4>
+        <p>The human contact details should enable the user to connect with the  organization or the part of the organization that can assist with the relevant content. For example, an online jobs / recruitment portal should provide a contact method for the team that supports the recruitment portal and not a catch-all for the entire company. Each layer of contact added prolongs the time before the user will receive help.</p>
+<h4>Mechanism for human contact</h4>
+        <p>The mechanism for human contact enables a person to express what they are looking for using their own words. For some with cognitive disabilities, this may be the best way for them to find an answer to their problem.</p>
+<h4>Self-help</h4>
+        <p>For pages for which no human support is available, a self-help option should say that no human support is available.  Self-help options should go beyond allowing the user to search within the site. Contextual help is still recommended (see <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/minimize-error-context-help.html">3.3.5 Help</a> for more information), but a self-help option should provide a single location that makes it easier for people with cognitive disabilities to understand what help is available without having to hunt for it. While some people may easily be able to identify that no support would be available for a particular type of Web site, this may not be apparent to some users with disabilities.</p>
+<h4>Automated</h4>
         <p>If a chatbot is provided, it should meet other WCAG Success Criteria for your conformance level. Chatbots which work more effectively for everyone, and particularly for people with cognitive disabilities should:</p>
 
 			<ul>
-            <li>recognize misspelled words,</li>
-            <li>provide human contact details if the chatbot is unable to provide a satisfactory response after 3 attempts, and</li>
+            <li>recognize misspelled words;</li>
+            <li>provide human contact details if the chatbot is unable to provide a satisfactory response after 3 attempts; and</li>
             <li>be dismissed with a single interaction, and recalled using a link or button.</li>
          </ul>
 
         <p>It is not the intent of this Success Criterion to require authors to provide help information on PDFs or other static documents that may be available for viewing/download from the Web pages. It is also not the intent to require contact information if:</p>
          <ul>
-            <li>A Web site is not supported,</li>
-				<li>Content is archived, or </li>
-            <li>When finding help would invalidate the activity such as in a testing situation. </li>
+            <li>a Web site is not supported;</li>
+				<li>content is archived; or </li>
```

Comment:
remove stay tabs?

Reply:
I didn't intentionally do any clean up of code spacing where it didn't affect output.